src/pages/lib/in-app-messaging/display-message/q/platform/[platform].mdx

```diff
@@ -1,8 +1,12 @@
 export const meta = {
   title: `Display message`,
-  description: `Learn how In-app messages are displayed when an In-App Messaging or analytics event is sent and matches the criteria set forth by your active In-App Messaging campaigns.`
+  description: `Learn how in-app messages are displayed when an In-App Messaging or analytics event is sent and matches the criteria set forth by your active In-App Messaging campaigns.`
```

Comment:
Interesting case change. So, if I'm interpreting this correctly "in-app messages" are a lower case thing the feature deals with where "In-App Messaging" is the proper case title of the feature. Do we apply this consistently through the documentation?

Reply:
Quick survey of this change makes it look like you have made this uniform.  Will cross check when I get the docs up in my review tomorrow morning.

Reply:
The way I've approached the feature has been:
- *In-App Messaging* is the feature - title cased
- *in-app message(s)* are just items/concepts the feature leverages - just regular lower-cased English